core/modules/ip_checksum.cc

```diff
@@ -34,6 +34,8 @@
 #include "../utils/ether.h"
 #include "../utils/ip.h"
 
+enum { DEFAULT_GATE = 0, _DROP_GATE };
```

Comment:
`_DROP_GATE` could get easily confused with `DROP_GATE` used by the scheduler to discard packets. Perhaps rename these to `VALID_GATE` (or `DEFAULT_GATE`) and `INVALID_GATE`?

Reply:
`INVALID_GATE` is also already defined. Renamed the gates.